third_party/versions.bzl

```
@@ -102,7 +102,7 @@ MAVEN_TEST_DEPENDENCY_VERSIONS = {
     "androidx.compose.ui:ui-test-junit4": "1.1.1",
     "androidx.test.espresso:espresso-accessibility": "3.1.0",
     "androidx.test.espresso:espresso-contrib": "3.1.0",
-    "androidx.test.espresso:espresso-core": "3.2.0",
+    "androidx.test.espresso:espresso-core": "3.5.0-alpha05",
```

Could we use the stable 3.5.0? At the time of creating the issue we were using an older version of Kotlin which was not compatible, but now it might work. Please try that and let me know.

```suggestion
    "androidx.test.espresso:espresso-core": "3.5.0,
```

---

done